enhancements/ingress/ingress-operator-operand-logging-level.md

```diff
@@ -80,17 +74,50 @@ The Ingress Operator currently uses [Zapr](https://github.com/go-logr/zapr) logg
 are displayed in the Ingress Operator pod logs. Debug-level logs are currently omitted in the Ingress Operator pod logs, although, the Ingress Operator itself rarely logs at any level
 other than `log.Info` and `log.Error`. In general, more lower-level logging calls could be added to the Ingress Operator.
 
-For this enhancement, a new `OperatorLogLevel` field is added to the Ingress Config resource:
+In a previous iteration of this enhancement proposal, the `OperatorLogLevel` field was proposed to be included in the
+Ingress Config resource (ingresses.config.openshift.io). However, the intent of the Ingress Config resource is to define
+cluster-wide ingress policies and configuration, not to define specific configuration for the Ingress Operator. As such,
+a new Custom Resource Definition (CRD) must be defined for Ingress Operator configuration, called `Ingress`
+(ingresses.operator.openshift.io):
 
 ```go
-type IngressSpec struct {
-	// <snip>
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+// +kubebuilder:object:root=true
+//
+// Ingress contains configuration options specific to the Ingress Operator itself.
+//
+// Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer).
+// +openshift:compatibility-gen:level=1
+type Ingress struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec is the specification of the desired behavior of the Ingress Operator.
+	Spec IngressSpec `json:"spec,omitempty"`
+	// status is the most recently observed status of the Ingress Operator.
+	Status IngressStatus `json:"status,omitempty"`
+}
+```
 
-	// operatorLogLevel controls the logging level of the Ingress Operator.
-	// See LogLevel for more information about each available logging level.
+The new resource will be cluster scoped. To keep consistent with the naming conventions for other cluster-scoped
+resources where only a single instance is expected, the default name will be "cluster".
+
+The `OperatorLogLevel` field will be included in the `spec` of the Ingress CRD:
+
+```go
+type IngressSpec struct {
+	// operatorLogLevel is the log level of the ingress operator. Options are:
+	// * Normal
+	// * Debug
+	// * Trace
+	// * TraceAll
```

Comment:
That doesn't seem to map well to neither of the two. Why should we use with the logging levels defined in `config.operator.openshift.io` then?